### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by SamBozman** [dglent/meteo-qt#124] [Q&A]
Can someone explain how to install this weather app? I have a Linux Mint 20.2 installation and I have the latest Python 3 installed as well. Can someone walk me through installing meteo-qt-master 3.1 currently sitting in my download folder?
(The directions on this site do not make any sense to me)

Thanks for your help
Sam

**Comment by dglent** [comment]
You can install it as root with the command:
(maybe in your system "python" executes python 3 so you don't need to run the command "python3" (but "python")
python3 setup.py install

Or without install it you can run:
(assuming you are at the first level in the folder)
python3 meteo_qt/meteo_qt.py
